libs/ui/src/components/NeoButton/NeoButton.scss

```diff
@@ -120,6 +120,23 @@
   &.no-shadow {
     box-shadow: none;
   }
+
+  &--rounded {
+    border-radius: 9999px;
```

Comment:
why this number?

Reply:
following oruga default value https://github.com/oruga-ui/oruga/blob/f01f5917d58646b96b2d6268931357b1727b992d/packages/oruga/src/scss/utilities/_variables.scss#L6